### PR TITLE
Fixture/update config files

### PR DESCRIPTION
**Pull request by steven-heng**
Npm audit found 1 low vulnerabilities
Modify rules command of package.json and Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
 	echo sonar.projectVersion=`cat package.json | python -c "import json,sys;obj=json.load(sys.stdin);print obj['version'];"` >> sonar-project.properties
 	wget https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-2.8.zip
 	unzip sonar-scanner-2.8.zip
 ifdef CI_PULL_REQUEST
-	@sonar-scanner-2.8/bin/sonar-runner -e -Dsonar.analysis.mode=preview -Dsonar.github.pullRequest=${shell basename $(CI_PULL_REQUEST)} -Dsonar.github.repository=$(REPO_SLUG) -Dsonar.github.oauth=$(GITHUB_TOKEN) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS) -Dsonar.host.url=$(SONAR_HOST_URL)
+	@sonar-scanner-2.8/bin/sonar-scanner -e -Dsonar.analysis.mode=preview -Dsonar.github.pullRequest=${shell basename $(CI_PULL_REQUEST)} -Dsonar.github.repository=$(REPO_SLUG) -Dsonar.github.oauth=$(GITHUB_TOKEN) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS) -Dsonar.host.url=$(SONAR_HOST_URL)
 endif
 ifeq ($(CIRCLE_BRANCH),develop)
 	@sonar-scanner-2.8/bin/sonar-runner -e -Dsonar.analysis.mode=publish -Dsonar.host.url=$(SONAR_HOST_URL) -Dsonar.login=$(SONAR_LOGIN) -Dsonar.password=$(SONAR_PASS)
```
Comment (jkernech) [inline Makefile]: Use `sonar-scanner` rather `sonar-runner` since it's deprecated

**Review by jkernech — CHANGES_REQUESTED** [review]

**Review by jkernech — CHANGES_REQUESTED** [review]
Tests need to be fixed

File: src/index.js [hunk 2]
```diff
@@ -36,52 +36,52 @@ class Logentries extends stream.Subscriber {
...
-    @param message {Object} - message package object
-    @see LoggerStream message package object structure description
-  **/
+   * @function handle
+   * Process a message and log it if the subscriber is switched on and message's log level is >= than MIN_LOG_LEVEL
+   * Finally, pass the message to the next subscriber if any
+   * @param message {Object} - message package object
+   * @see LoggerStream message package object structure description
+   * */
   handle(message) {
```
Comment (dialonce) [inline src/index.js:72]: CRITICAL Refactor this function to reduce its Cognitive Complexity from 19 to the 15 allowed. [rule](https://sonar.dialonce.net/coding_rules#rule_key=javascript%3AS3776)

**Comment by dialonce** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**